### ISSUE TITLE
The naming of my isoforms is different from what is mentioned in your manual.

### ISSUE THREAD

**Issue by gnilihzeux** [BrooksLabUCSC/flair#333]
Dear author,

**I have read the paper (https://doi.org/10.1038/s41467-020-15171-6)  
and the manual (https://flair.readthedocs.io/en/latest/) and I still have a question about**

Aftter **flair collapse**, the naming of my isoforms in the GTF is different from what is mentioned in your manual.

> If multiple TSSs/TESs are allowed (toggle with --max_ends or --no_redundant), then a -1 or higher will be appended to the end of the isoform name for the isoforms that have identical splice junction chains and differ only by their TSS/TES.

I found some transcripts contains -0, which has no other isoforms.

I sorted the table and list somes below.

```
ENST00000005178.6 | ENSG00000004799.8
ENST00000005178.6-0 | ENSG00000004799.8
ENST00000005180.9-0 | ENSG00000006606.9
ENST00000005257.7 | ENSG00000006451.8
ENST00000005257.7-1 | ENSG00000006451.8
ENST00000005259.9 | ENSG00000075790.12
ENST00000005259.9-1 | ENSG00000075790.12
ENST00000005260.9 | ENSG00000006453.14
ENST00000005260.9-0 | ENSG00000006453.14
ENST00000005340.10 | ENSG00000004975.12
ENST00000005374.10 | ENSG00000006625.18
ENST00000005386.8 | ENSG00000005175.10
ENST00000005386.8-0 | ENSG00000005175.10
ENST00000005558.8-0 | ENSG00000006652.15
```

**Comment by cafelton** [comment]
You're right, this is a bit weird. It should all be reflective of isoforms with the same splice junctions and different ends but I'll look into it.
